kirkstone-poky-script.bash

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash -x -e
+
+export PLATFORM_BRANCH="hmx-mickledore"
```

Comment:
This branch should not be used.

Reply:
This script was at one point the script used by the Jenkins job "kirkstone-poky". As the Jenkins scripts become more complicated, it could be a good idea to version control them here instead of in the Jenkins jobs. This way they can be run on your local machine.